src/lib/utils/config/src/statics.rs

```diff
@@ -1,21 +1,21 @@
-//! # Statics module.
-//!
-//! Contains the static global configuration and its related functions.
-
 use crate::server_config::ServerConfig;
+use dashmap::DashMap;
 use ferrumc_general_purpose::paths::get_root_path;
 use lazy_static::lazy_static;
 use std::fs::File;
 use std::io::{Read, Write};
 use std::process::exit;
 use tracing::{error, info};
+use uuid::Uuid;
 
 /// The default server configuration that is stored in memory.
 pub(crate) const DEFAULT_CONFIG: &str = include_str!("../../../../../.etc/example-config.toml");
 
 lazy_static! {
     /// The server configuration that is stored in memory.
     static ref CONFIG: ServerConfig = create_config();
+    /// The whitelist of player uuids.
+    static ref WHITELIST: DashMap<u128, String> = create_whitelist();
```

Comment:
Use a DashSet, not a DashMap. You can either just store the uuids written in the file as a string, or you can hash the username to a u128 and store it that way. Either way storing a username and uuid for each entry is redundant and means that in this case checking a username could have up to O(n) lookup, whereas storing uuids *and* usernames as the key gets you O(1) lookup.